src/core/ledger.js

```diff
@@ -47,11 +46,11 @@ Ledger.prototype.calcAccountRootEntryHash = function(account) {
  * @return {UInt256}
  */
 Ledger.calcOfferEntryHash =
-Ledger.prototype.calcOfferEntryHash = function(account, sequence) {
-  account = UInt160.from_json(account);
-  sequence = parseInt(sequence, 10);
+Ledger.prototype.calcOfferEntryHash = function(_account, _sequence) {
```

Comment:
I think the ideal solution is to use strict types and more descriptive names like `const sequence = parseInt(sequenceString, 10)`